crates/typst-kit/src/package.rs

```diff
@@ -1,7 +1,9 @@
 //! Download and unpack packages and package indices.
 
 use std::fs;
+use std::io::{self, Write};
 use std::path::{Path, PathBuf};
+use std::{thread::sleep, time::Duration};
```

Comment:
please use one import per module

Reply:
? You mean separate for `thread` and `time`? Also it's temporary. Also also it looks like I removed `sleep()` at some point so it's not even used. I think I will re-add the sleep for easier debugging. All debugging stuff will be removed when the WIP label is removed.

Reply:
Yeah, I meant thread and time. Basically not grouping multiple imports from different modules. But if it's temporary it doesn't matter.